lib/sqlalchemy/dialects/postgresql/base.py

```diff
@@ -3030,17 +3030,22 @@ def initialize(self, connection):
         # https://www.postgresql.org/docs/9.3/static/release-9-2.html#AEN116689
         self.supports_smallserial = self.server_version_info >= (9, 2)
 
-        std_string = connection.exec_driver_sql(
-            "show standard_conforming_strings"
-        ).scalar()
-        self._backslash_escapes = std_string == "off"
+        self.set_backslash_escapes()
 
         self._supports_drop_index_concurrently = self.server_version_info >= (
             9,
             2,
         )
         self.supports_identity_columns = self.server_version_info >= (10,)
 
+    def set_backslash_escapes(self):
```

Comment:
this should be _set_backslash_escapes

Reply:
Should I also move it further down to the end of the class with the rest of the private methods? Unsure if there's a convention around method ordering.

Reply:
not much, it's good enough